base/bucket_test.go

```diff
@@ -276,3 +275,19 @@ func TestGetPoolName(t *testing.T) {
 	fakeBucketSpec.PoolName = "Liverpool"
 	assert.Equal(t, "Liverpool", fakeBucketSpec.GetPoolName())
 }
+
+func TestGetViewQueryTimeoutMs(t *testing.T) {
+	fakeBucketSpec := &BucketSpec{}
+	assert.Equal(t, uint64(0x124f8), fakeBucketSpec.GetViewQueryTimeoutMs())
```

Comment:
Why hex here? `75000` is a lot more understandable

Reply:
Hex is included to avoid the calculation again while asserting the value; This has been changed to make it more readable.